simple_auth/lib/src/oauth/oauthAuthenticator.dart

```diff
@@ -5,11 +5,12 @@ class OAuthAuthenticator extends WebAuthenticator {
   String clientSecret;
   String tokenUrl;
   OAuthAuthenticator(String identifier, String clientId, this.clientSecret,
-      this.tokenUrl, String baseUrl, String redirectUrl) {
+      this.tokenUrl, String baseUrl, String redirectUrl, List<String> scopes) {
```

Comment:
Can we make the scopes optional? Not everything requires them. And then it is not a breaking change!

Reply:
So make them a named parameter??  That would work if that's ok?